test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -289,70 +288,140 @@ public void PlattCalibratorOnnxConversionTest()
+
+            // Step 3: Test calibrator with a non-default Score column name and without any binary prediction trainer
+            IDataView dataSoloCalibratorNonStandard = mlContext.Data.LoadFromEnumerable(GetCalibratorTestDataNonStandard());
+
+            var pipelineSoloCalibratorNonStandard = mlContext.BinaryClassification.Calibrators
+                .Platt(scoreColumnName: "ScoreX", slope: -1f, offset: -0.05f);
+            var onnxFileNameSoloCalibratorNonStandard = $"{pipelineSoloCalibratorNonStandard}-SoloCalibrator-NonStandard.onnx";
+
+            TestPipeline(pipelineSoloCalibratorNonStandard, dataSoloCalibratorNonStandard, onnxFileNameSoloCalibratorNonStandard, new ColumnComparison[] { new ColumnComparison("Probability", 3) });
+
+            Done();
+        }
+
+        [Fact]
+        public void NaiveCalibratorOnnxConversionTest()
+        {
+            // Step 1: Test calibrator with binary prediction trainer
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
+            foreach (var estimator in estimators)
+            {
+                var pipeline = initialPipeline.Append(estimator).Append(mlContext.BinaryClassification.Calibrators.Naive());
+                var onnxFileName = $"{estimator}-WithNaiveCalibrator.onnx";
+
+                TestPipeline(pipeline, dataView, onnxFileName, new ColumnComparison[] { new ColumnComparison("Score", 3), new ColumnComparison("PredictedLabel"), new ColumnComparison("Probability", 3) });
+            }
+
+            // Step 2: Test calibrator without any binary prediction trainer
+            IDataView dataSoloCalibrator = mlContext.Data.LoadFromEnumerable(GetCalibratorTestData());
+
+            var pipelineSoloCalibrator = mlContext.BinaryClassification.Calibrators
+                .Naive();
+            var onnxFileNameSoloCalibrator = $"{pipelineSoloCalibrator}-SoloCalibrator.onnx";
+
+            TestPipeline(pipelineSoloCalibrator, dataSoloCalibrator, onnxFileNameSoloCalibrator, new ColumnComparison[] { new ColumnComparison("Probability", 3) });
+
+            // Step 3: Test calibrator with a non-default Score column name and without any binary prediction trainer
+            IDataView dataSoloCalibratorNonStandard = mlContext.Data.LoadFromEnumerable(GetCalibratorTestDataNonStandard());
+
+            var pipelineSoloCalibratorNonStandard = mlContext.BinaryClassification.Calibrators
+                .Naive(scoreColumnName: "ScoreX");
+            var onnxFileNameSoloCalibratorNonStandard = $"{pipelineSoloCalibratorNonStandard}-SoloCalibrator-NonStandard.onnx";
+
+            TestPipeline(pipelineSoloCalibratorNonStandard, dataSoloCalibratorNonStandard, onnxFileNameSoloCalibratorNonStandard, new ColumnComparison[] { new ColumnComparison("Probability", 3) });
+
             Done();
         }
 
-        class PlattModelInput
+        class ModelInput
```

Comment:
**Nit:** Please, rename this and the other class to `CalibratorInput` and `CalibratorInputNonStandard` for clarity.